images-list

```diff
@@ -34,3 +34,8 @@ us.gcr.io/k8s-artifacts-prod/external-dns/external-dns rancher/kubernetes-extern
 openpolicyagent/gatekeeper rancher/opa-gatekeeper v3.1.0-beta.11
 k8s.gcr.io/k8s-dns-node-cache rancher/k8s-dns-node-cache 1.15.7
 sonobuoy/sonobuoy rancher/sonobuoy-sonobuoy v0.16.3
+banzaicloud/logging-operator rancher/logging-operator 3.4.0
+jimmidyson/configmap-reload rancher/configmap-reload v0.2.2
+paynejacob/fluent-bit-out-syslog rancher/fluent-bit-out-syslog latest
+banzaicloud/fluentd rancher/fluentd v1.10.4-alpine-2
```

Comment:
We already have a fluentD repo but its custom, so this one should also be org-namespaced. It looks like banzai's is different image entirely than the official fluentD ? 
* https://hub.docker.com/_/fluentd?tab=tags
* https://hub.docker.com/r/banzaicloud/fluentd/tags
* https://hub.docker.com/r/rancher/fluentd/tags

Reply:
They may be compatible but this image has all the plugins installed that the operator supports so I think it is worth having 2.